python/GafferSceneUI/LightEditor.py

```diff
@@ -155,6 +152,16 @@ def registerParameter( cls, rendererKey, parameter, section = None, columnName =
 			assert( isinstance( parameter, IECoreScene.ShaderNetwork.Parameter ) )
 			parameter = ( parameter.shader, parameter.name )
 
+		return parameter
```

Comment:
This would be more succinct by returning above, instead of assigning to `parameter` and returning that.

Reply:
Done in https://github.com/GafferHQ/gaffer/pull/5891/commits/6da2d7c7c6aa3c84051a74608b1d9096c6fa826c